generate_binary.py

```diff
@@ -0,0 +1,114 @@
+#! /usr/bin/env python
```

Comment:
I'm not sure that it should called "binary".

May be "package"... or "executable"

Reply:
Maybe `compile_executable`?

Reply:
Yes, it's better IMO